pkg/kubevirt/machine_test.go

```diff
@@ -246,6 +246,7 @@ var _ = Describe("With KubeVirt VM running", func() {
 		externalMachine, err := defaultTestMachine(machineContext, fakeClient, fakeVMCommandExecutor, []byte(sshKey))
 		Expect(err).NotTo(HaveOccurred())
 		providerId, err := externalMachine.GenerateProviderID()
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
just nit: `NotTo` or `ToNot`? probably need to have one convention

Reply:
ginkgo support them all + `ShouldNot`